src/components/postCompletionViews/PostCompletionView.stories.js

```diff
@@ -0,0 +1,19 @@
+import Body from 'components/Body';
+
+import PostCompletionView from './PostCompletionView';
+
+export default {
+  title: 'Private API / Post completion views ',
+  render: ({body, ...args}) => <PostCompletionView {...args} body={<Body>{body}</Body>} />,
+};
```

Comment:
```suggestion
export default {
  title: 'Private API / Post completion views ',
  component: PostCompletionView,
  render: ({body, ...args}) => <PostCompletionView {...args} body={<Body>{body}</Body>} />,
};
```
this helps with auto-generated documentation when documenting the props etc.